public/css/concept-page.css

```
@@ -118,26 +128,456 @@
+.concept-page-b2__route-item.is-active .concept-page-b2__route-marker,
+.concept-page-b2__route-marker-item.is-active .concept-page-b2__route-marker {
+  background: rgba(var(--lavender-500-rgb), 0.72);
+  border-color: rgba(var(--lavender-500-rgb), 0.88);
+  box-shadow: 0 0 0 5px rgba(var(--lavender-500-rgb), 0.10);
+}
+.concept-page-b2__route-copy {
+  display: grid;
+  gap: 4px;
+}
+.concept-page-b2__route-title {
+  color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)